activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/src/main/java/org/activiti/cloud/services/events/listeners/CloudTimerCanceledProducer.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.events.listeners;
+
+import org.activiti.api.process.model.events.BPMNTimerCanceledEvent;
+import org.activiti.api.process.runtime.events.listener.BPMNElementEventListener;
+import org.activiti.cloud.services.events.converter.ToCloudProcessRuntimeEventConverter;
+
+public class CloudTimerCanceledProducer implements BPMNElementEventListener<BPMNTimerCanceledEvent> {
```

Comment:
with usage of generics as mentioned in a previous comment, could we have a single producer class?

Reply:
@ffazzini that's a good point. I think we could simplify it, but it would require a bigger refactoring involving all the producers (not only timers). So I propose to don't to it now.